Common.xml

```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
       </Item>
       
       <Item ID="6044">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Battery Percentage</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
+        <Units>%</Units>
```

Comment:
Currently “%” is for values from 0 to 1. I’d recommend leaving this empty for now until we register a proper unit for percentage 0 to 100.

Reply:
Agree